todoman/model.py

```diff
@@ -297,7 +299,12 @@ def set_field(self, name, value):
         self.vtodo.pop(name)
         if value:
             logger.debug("Setting field %s to %s.", name, value)
-            self.vtodo.add(name, value)
+            if name == 'categories':
+#                v = icalendar.prop.vInline(','.join(value))
+                v = self.vtodo.set_inline(name, value)
```

Comment:
Looks like this needs to be `if value is not None`.

Reply:
set_inline in the icalender library is defined here:
https://github.com/collective/icalendar/blob/master/src/icalendar/cal.py#L256